### DISCUSSION TITLE
Welcome to intellij-erlang Discussions!

### DISCUSSION THREAD

**Discussion by ignatov** [ignatov/intellij-erlang#932] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by kvakvs** [comment]
Looking to contribute at small or large scale to the project. There are multiple things which could be improved, for sure. There are also many things which I am not using for various reasons (like running from IDE, debugging, rebar support).

My main problems are lack of (personal) time, as this is not sponsored by any organisation and I cannot cut a share of my working hours to spend more time on this project, and also lack of understanding of the IntelliJ plugin API. So every encounter with the plugin source code is a painful struggle.

**Comment by alexei-salikhov** [comment]
Hi, can you switch off "parameter name hints for function calls", they take too much space and don't add any value for me personally. We already have specs.
